_posts/0000-01-02-SibinBinu.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: Slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
### _**If it were already in ones head without giving a fuss about learning things!**_